draft-ietf-quic-transport.md

```diff
@@ -1242,33 +1242,26 @@ Implementations SHOULD ensure that peers have the requested number of connection
 IDs available to reduce the possibility of peers exhausting their supply of
 available connection IDs.  An implementation could do this by always supplying a
 new connection ID for each connection ID retired with a CONNECTION_ID_FINISHED
-frame.
+frame. When a receiver of a packet notices that its peer is now using a
+previously unused connection ID, it MAY choose to supply its peer with a new
+connection ID using a NEW_CONNECTION_ID frame to reduce the possibility of its
+peer running out of available connection IDs.
 
-When a receiver of a packet notices that its peer is now using a previously
-unused connection ID, it MAY choose to supply its peer with a new connection ID
-using a NEW_CONNECTION_ID frame to reduce the possibility of its peer running
-out of available connection IDs.
-
-It its peer selected non-zero-length connection IDs, an endpoint that receives a
-packet with a previously unused connection ID, SHOULD switch to sending with a
-different connection ID from its set of valid connection IDs when sending
-responses.  This can help to ensure that when an endpoint migrates to a new path
-or changes connection ID on an existing path, the packets will use different
-connection IDs in both directions. Likewise, when an endpoint receives a
-CONNECTION_ID_FINISHED frame, it SHOULD retire any connection IDs that were used
-in packets generated in response to that connection ID.
+An endpoint that receives a packet with a previously unused connection ID,
+SHOULD also switch to sending with a different connection ID.  This can help to
+ensure that different connection IDs will be used in both directions when an
+endpoint migrates to a new path or changes connection ID on an existing path.
 
 
 ### Consuming Connection IDs
 
 If an endpoint's peer has selected a non-zero-length connection ID, the endpoint
 maintains a set of connection IDs received from the peer that it can use when
-sending packets.  The initial connection ID used when sending packets is sent by
-the peer as the Source Connection ID during the handshake. To ensure that
-sufficient connection IDs are available for future use, the endpoint
-SHOULD request additional connection IDs using the `requested_connection_ids`
-transport parameter.  The endpoint will then receive up to the requested number
-of connection IDs via NEW_CONNECTION_ID frames from its peer.
+sending packets. To ensure that sufficient connection IDs are available for
+future use, the endpoint SHOULD request additional connection IDs using the
+`requested_connection_ids` transport parameter.  The endpoint will then receive
```

Comment:
Are we dropping this transport parameter or not?

Reply:
Yup, looks like there's no objection to doing that, so I'm removing it. Just wanted to double check on that one.